src/coverage-system/coverageservice.ts

```diff
@@ -23,6 +24,7 @@ enum Status {
 }
 
 export class CoverageService {
+    public statusBar: StatusBarToggler;
```

Comment:
I would probably leave this private unless you think we need to expose this to the callers of the coverage service?